services/predictors/layoutparser/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# dev image for running verification checks
+FROM python:3.8 AS dev
+
+RUN apt-get update && apt-get install nginx iproute2 python3-opencv -y
+WORKDIR /opt/ml/code
+
+COPY services/predictors/layoutparser/PROJECT_NAME.txt .
+COPY services/predictors/layoutparser/requirements.txt .
+COPY services/predictors/layoutparser/setup.py .
+
+RUN pip install -r requirements.txt
+RUN pip install --upgrade .[dev]
+RUN pip install "git+https://github.com/facebookresearch/detectron2.git@v0.4#egg=detectron2"
+
+COPY services/predictors/layoutparser .
+COPY mmda mmda
+ENTRYPOINT ["/bin/bash"]
+
+
+# server image for running the API
```

Comment:
Using multi-stage build so that we can have a lighter-weight container (with no model downloaded) for format-checks, etc.